bridge_adaptivity/api/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from module.models import Activity, Collection
+
+
+class ActivitySerializer(serializers.ModelSerializer):
+    """
+    Serializer for Activity API view set.
+
+    Serializer is based on the Activity model's fields.
+    """
+
+    class Meta:
+        model = Activity
+        fields = '__all__'
```

Comment:
is it secure to allow to serialize all fields?

Reply:
We have not any sensitive data in the Activity model.